project/src/pages/property-page/near-places-list/near-places-list.tsx

```diff
@@ -10,7 +15,7 @@ export function NearPlacesList(props: {nearData: IOffer[]}) {
 
         <div className="near-places__list places__list">
           {
-            props.nearData.map((place) => <PlaceCard key={place.id} offer={place}/>)
+            props.nearData.map((place) => <PlaceCard key={place.id} offer={place} onPlaceCardHover={props.onPlaceCardHover}/>)
```

Comment:
и тут уже сразу вызывать `onPlaceCardHover`
```suggestion
            props.nearData.map((place) => <PlaceCard key={place.id} offer={place} onPlaceCardHover={onPlaceCardHover}/>)
```